app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invaild command format")
```

Comment:
There is a typo in the error message: 'Invaild' should be corrected to 'Invalid'.